### DISCUSSION TITLE
Not able to build Ghostty from Packaging.md

### DISCUSSION THREAD

**Discussion by dhrm1k** [ghostty-org/ghostty#3303] [Q&A] [answered]
Going through [Packaging.md](https://github.com/ghostty-org/ghostty/blob/main/PACKAGING.md) and not able to build Ghostty. Getting this error.

![Image](https://github.com/user-attachments/assets/7d8c0bc8-d21c-4c7e-b0fb-b2543892d26b)

**Comment by 00-kat** [comment, marked as answer]
Is there any other output?  That screenshot says "building Ghostty failed :(".  And also, why are you building as root...?
